packages/extension/src/background/index.ts

```diff
@@ -1,5 +1,5 @@
 import 'content-scripts-register-polyfill';
-import { browser, Runtime, Tabs } from 'webextension-polyfill-ts';
+import browser, { runtime, Runtime, Tabs } from 'webextension-polyfill';
```

Comment:
Just use `browser.runtime`? It will be easier to refactor down the line if we change the polyfill since `browser` is usually available as a global in extension context.

Reply:
I see you use `browser` top level in all other places so this is the only different case to adjust.

Reply:
good catch, updated 🙏